.coafile

```diff
@@ -12,12 +12,22 @@ use_spaces = True
 # will be executed sequentially; also we need the LineLengthBear to double
 # check the line length because PEP8Bear sometimes isn't able to correct the
 # linelength.
-bears = SpaceConsistencyBear, PyUnusedCodeBear, QuotesBear
+bears = SpaceConsistencyBear, QuotesBear
 language = Python
 preferred_quotation = '
 
 default_actions = *: ApplyPatchAction
 
+[python_imports]
+# PyUnusedCodeBear is run separately beacause we need to check for unused 
+# imports. Do not set default_action to ApplyPatchAction as it may lead to some 
```

Comment:
The first line doesn't really add any new information that I can't understand from reading the lines below. Just keep the second line.